cogeo_mosaic/mosaic.py

```diff
@@ -138,7 +138,7 @@ def _create_mosaic(
         tree = STRtree(dataset_geoms)
 
         for tile in tiles:
-            quadkey = mercantile.quadkey(tile)
+            quadkey = str(mercantile.quadkey(tile))
```

Comment:
FIX, this should be a string (per the specification)

Reply:
It already is a string.